vpr/src/pack/cluster.cpp

```diff
@@ -400,7 +431,8 @@ std::map<t_logical_block_type_ptr, size_t> do_clustering(const t_packer_opts& pa
                                                          std::vector<t_lb_type_rr_node>* lb_type_rr_graphs,
                                                          const t_ext_pin_util_targets& ext_pin_util_targets,
                                                          const t_pack_high_fanout_thresholds& high_fanout_thresholds,
-                                                         AttractionInfo& attraction_groups) {
+                                                         AttractionInfo& attraction_groups,
+                                                         bool& floorplan_regions_overfull) {
```

Comment:
mention the bool floorplan_regions_overfull in the comment for the do_clustering, because it's like an additional return value. Same for the attraction groups